test/index.test.js

```diff
@@ -72,9 +72,9 @@ describe('Vero', function() {
         analytics.stub(window._veroq, 'push');
       });
 
-      it('should push "trackPageview"', function() {
+      it('should push "viewed_page"', function() {
         analytics.page();
-        analytics.called(window._veroq.push, ['trackPageview']);
+        analytics.called(window._veroq.push, ['track', 'viewed_page', {}, { source: 'segment' }]);
```

Comment:
`page.properties()` returns the current page call's properties object. By default our `.page()` call collects some context page metadata:

```
{
	      "path": "/context.html",
	      "referrer": "http://localhost:9876/?id=70213249",
	      "search": "",
	      "title": "",
	      "url": "http://localhost:9876/context.html"
	    },
```

☝️ is what gets set inside the test suite. So you should change the `analytics.called()` method to include these properties!